### PR TITLE
feat(boards): Add nice!60 board definition

### PR DESCRIPTION
**Pull request by Nicell**
Adds the board definition for the nice!60, a stocked 60% PCB up for sale in the coming weeks. https://nicekeyboards.com/nice-60

### CODE REVIEW

File: app/boards/arm/nice60/nice60.dts [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+		spi-one-frame = <0x70>;
+		spi-zero-frame = <0x40>;
+	};
+};
+
+&uart0 {
+	compatible = "nordic,nrf-uarte";
+	status = "okay";
+	current-speed = <115200>;
+	tx-pin = <14>;
```
Comment (petejohanson) [inline app/boards/arm/nice60/nice60.dts:133]: Do you in fact have a serial breakout on this?

Comment (Nicell) [reply app/boards/arm/nice60/nice60.dts:133]: There is not. Does removing this no longer cause USB logging to fail?

Comment (petejohanson) [reply app/boards/arm/nice60/nice60.dts:133]: It shouldn't. I believe I cherry picked the fix from upstream to make this not required any more.

Comment (Nicell) [reply app/boards/arm/nice60/nice60.dts]: I remember this now, removed.

File: app/boards/arm/nice60/nice60.keymap [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+// | CAPS  |  A  |  S  |  D  |  F  |  G  |  H  |  J  |  K  |  L  |  ;  |  '  |     ENTER    |
+// |  SHIFT  |  Z  |  X  |  C  |  V  |  B  |  N  |  M  |  ,  |  .  |  /  |      SHIFT       |
+// |  CTL  |  WIN  |  ALT  |            SPACE              |  ALT  |  WIN  |  MO(1) |  CTL  |
+// ------------------------------------------------------------------------------------------
+			bindings = <
+	&gresc  &kp NUM_1 &kp NUM_2 &kp NUM_3 &kp NUM_4 &kp NUM_5 &kp NUM_6 &kp NUM_7 &kp NUM_8 &kp NUM_9 &kp NUM_0 &kp MINUS &kp EQL  &kp BKSP
+	&kp TAB  &kp   Q   &kp   W   &kp   E   &kp   R   &kp   T   &kp   Y   &kp   U   &kp   I   &kp   O   &kp   P   &kp LBKT &kp RBKT &kp BSLH
+	&kp CLCK  &kp   A   &kp   S   &kp   D   &kp   F   &kp   G   &kp   H   &kp   J   &kp   K   &kp   L   &kp  SCLN  &kp  QUOT        &kp RET
+	&kp LSFT   &kp   Z   &kp   X   &kp   C   &kp   V   &kp   B   &kp   N   &kp   M   &kp   CMMA   &kp   DOT   &kp   FSLH           &kp RSFT
+	&kp LCTL &kp   LGUI   &kp   LALT                     &kp SPC                         &kp   RALT    &kp   RGUI     &mo 1      &kp   RCTL
```
Comment (innovaker) [inline app/boards/arm/nice60/nice60.keymap:23]: Dude, some of these are out of date.

**Review by innovaker — CHANGES_REQUESTED** [review]
Thanks @Nicell!

Nothing particularly odd stood out except the deprecated codes.

Please also check:
- The copyright/license headers seem inconsistent?
- Copyright should be 2021.
- There's no `README.md`?

plus some nickpicking.  Pun intended.

I appreciate you've chosen to bypass `setup.ps1`, `setup.sh`, `build.yml` and `hardware.md` for now.

File: app/boards/arm/nice60/board.cmake [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier: MIT
+
+board_runner_args(nrfjprog "--nrf-family=NRF52" "--softreset")
+include(${ZEPHYR_BASE}/boards/common/blackmagicprobe.board.cmake)
+include(${ZEPHYR_BASE}/boards/common/nrfjprog.board.cmake)
```
[the remark this answers is not in the document]
Comment (Nicell) [reply app/boards/arm/nice60/board.cmake]: Updated to tested probes

**Review by innovaker — APPROVED** [review]
Thanks for following up and making the changes @Nicell!  Much appreciated 😃

**Review by petejohanson — APPROVED** [review]

**Pull request merged**